### PR TITLE
Bugfix: Import electrum multisig wallet (with available seed)

### PR DESCRIPTION
**Pull request by relativisticelectron**
1. A space was in recv_descriptor
2. hw_type key does not exist in the electrum wallet dict if the seed exists in the electrum wallet

How to reproduce the bug:  Create a 2/2 testnet electrum multisig wallet (with electrum seeds).  The file will look like:
```
{
    "addr_history": {

    },
    "addresses": {
        "change": [
        ],
        "receiving": [
        ]
    },
    "fiat_value": {},
    "frozen_coins": {},
    "invoices": {},
    "labels": {},
    "payment_requests": {},
    "prevouts_by_scripthash": {....},
    "seed_version": 41,
    "spent_outpoints": {......
    },
    "transactions": {.....    },
    "tx_fees": {.....    },
    "txi": {},
    "txo": {.....    },
    "use_encryption": false,
    "verified_tx3": {},
    "wallet_type": "2of2",
    "x1/": {
        "derivation": "m/1'",
        "pw_hash_version": 1,
        "root_fingerprint": "....",
        "seed": ".....",
        "seed_type": "segwit",
        "type": "bip32",
        "xprv": "...",
        "xpub": "..."
    },
    "x2/": {
        "derivation": "m/1'",
        "pw_hash_version": 1,
        "root_fingerprint": "...",
        "type": "bip32",
        "xprv": null,
        "xpub": "..."
    }
}
```

### CODE REVIEW

File: src/cryptoadvance/specter/util/wallet_importer.py [hunk 1]
```diff
@@ -266,7 +266,16 @@ def parse_wallet_data_import(cls, wallet_data):
...
-                cosigners_types.append({"type": d["hw_type"], "label": d["label"]})
+                if "hw_type" in d:
+                    cosigners_types.append({"type": d["hw_type"], "label": d["label"]})
+                else:  # this can occcur if no hardware wallet was used, but the seed is available
+                    cosigners_types.append(
+                        {"type": "electrum", "label": f"Electrum Multisig {i}"}
+                    )
+                    if "seed" in d:
+                        logger.warning(
+                            "The Electrum wallet contains a seed. The seed will not be imported."
```
Comment (relativisticelectron) [inline src/cryptoadvance/specter/util/wallet_importer.py:277]: Problem:  This warning disappears extremely fast. However I think it is important that the user realizes that the seed was not imported. How can I make this more visible for the user?

**Review by relativisticelectron — COMMENTED** [review]
Any suggestions?

File: src/cryptoadvance/specter/util/wallet_importer.py (hunk 1 above)
Comment (relativisticelectron) [inline src/cryptoadvance/specter/util/wallet_importer.py:277]: This warning is not visible for the user. However I think it is important that a user is warned. How can I show this warning to the user?

**Review by relativisticelectron — COMMENTED** [review]
How can I show this warning to the user?

File: src/cryptoadvance/specter/util/wallet_importer.py (hunk 1 above)
Comment (k9ert) [reply src/cryptoadvance/specter/util/wallet_importer.py:277]:
```suggestion
                        flash(_("The Electrum wallet contains a seed. The seed will not be imported."), "warning")
                            
```
This should work!

**Comment by k9ert** [comment]
LGTM but can you check the code-style? That's red right now. See:
https://docs.specter.solutions/desktop/development/#code-style

**Comment by relativisticelectron** [comment]
> Can you check the code-style? That's red right now. See: https://docs.specter.solutions/desktop/development/#code-style

Hi @k9ert :  The black coding style problem comes from the file: src/cryptoadvance/specter/managers/config_manager.py	   However I never touched this file.

File: src/cryptoadvance/specter/util/wallet_importer.py (hunk 1 above)
Comment (relativisticelectron) [reply src/cryptoadvance/specter/util/wallet_importer.py]: Great. Thanks, I did this and added the missing imports.

**Pull request merged**